Adding docs for userroles and permissions on user management dashboard. (#752)
supertokens/docs: v2/emailpassword/user-roles/delete-roles.mdx

You can delete any role you have created, if the role you are trying to delete does not exist then this has no  effect

:::tip

> You can easily delete any role that you have created from the user management dashboard. To know more about how to manage your user roles and permissions from user management dashboard look <a href="/docs/userdashboard/managing-user-roles-and-permissions" target="_blank">here</a>

Reviewer: ```suggestion
You can also delete roles from the user management dashboard. To know more about how to manage your user roles and permissions from user management dashboard see <a href="/docs/userdashboard/managing-user-roles-and-permissions" target="_blank">this page</a>.
```
